api/camunda/camunda-openapi.yaml

```diff
@@ -841,6 +841,48 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
 
+  /flownode-instances/search:
```

Comment:
@pepopowitz, this has been added by [a PR](https://github.com/camunda/camunda/pull/21783) that hasn't been reviewed by you and your team. I have notified the responsible devs again that this should happen in monorepo PRs.

Reply:
If you see any resources on your end to automate including you and your team in PR reviews automatically when the OpenAPI file is changed, let me know. We currently have too much release work coming up to tackle this (but I assume you do as well, so 🤷).